src/main/java/edu/kis/powp/command/ComplexCommandFactory.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.powp.command;
+
+import edu.kis.legacy.drawer.panel.DefaultDrawerFrame;
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDrawerAdapter;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class ComplexCommandFactory {
+    public static ComplexCommand getSquare(DrawPanelController drawPanelController, LineDrawerAdapter.lineTypeEnum lineType){
+        ComplexCommand commands = new ComplexCommand();
+        LineDrawerAdapter drawer = new LineDrawerAdapter(drawPanelController, lineType);
```

Comment:
Fixed driver instance. Commands should be able to use the current driver (i.e., DriverManager.getCurrentDriver())